woocommerce/woocommerce-android
# Remove Sort, Filters, bottom navigation bar and Add Product CTA from Product search

Closes: #3259 

Or at least most of it.


https://user-images.githubusercontent.com/2722505/138733215-3204d797-b2ad-4b52-b3b2-610e16571226.mov



### Description
This PR hides the bottom navigation bar, the "Sort", "Filters" and "Add Product" buttons when searching products.

The original description also mentions removing the search icon, but the icon is not removed when searching orders either. I thought it would be better to tackle that in a separate PR.

## Changes
- Updated ProductListFragment to show/hide the bottom toolbar according to the status of the view model
- Updated the logic that handled the visibility of the Add Product button and the sort and filter options in the view model 

### Testing instructions
1. Navigate to the Products tab
2. Select search. 
3. Search for products, cancel the search, see that the buttons behave as expected (see #3259 )
